browser/ui/views/tabs/brave_compound_tab_container.cc

```diff
@@ -529,16 +528,19 @@ void BraveCompoundTabContainer::ScrollTabToBeVisible(int model_index) {
   // Unfortunately, ScrollView's API doesn't work well for us. So we manually
```

Comment:
@sangwoo108 Could you please clarify what you actually meant by `ScrollView's API doesn't work well for us`,  Now I've used `ScrollView::ScrollToOffset` to invoke `ScrollView::OnScrolled` which  fixes this issue, but couldn't  identify any other breakings yet.

Reply:
It seems that ScrollToOffset() was private method and recently changed public. https://chromium-review.googlesource.com/c/chromium/src/+/4375909

Reply:
Could you remove the comment?